assets/src/components/detoursTable.tsx

```diff
@@ -16,27 +16,29 @@ export const DetoursTable = ({ data }: DetoursTableProps) => {
     <Table hover className="c-detours-table">
       <thead>
         <tr>
-          <th>Route</th>
-          <th className="u-hide-for-mobile">Starting Intersection</th>
-          <th className="u-hide-for-mobile">On detour since</th>
+          <th className="px-3 py-4">Route and direction</th>
+          <th className="px-3 py-4 u-hide-for-mobile">Starting Intersection</th>
+          <th className="px-3 py-4 u-hide-for-mobile">On detour since</th>
         </tr>
       </thead>
       <tbody>
         {data.map((detour, index) => (
           <tr key={index}>
-            <td className="align-middle">
-              <div className="c-detours-table__route-info-cell">
+            <td className="align-middle p-3">
+              <div className="d-flex">
                 <RoutePill routeName={detour.route} />
-                <div className="c-detours-table__route-info-text">
-                  <span>{detour.direction}</span>
-                  <span className="fw-bold">{detour.name}</span>
+                <div className="c-detours-table__route-info-text d-inline-block">
+                  <div className="pb-1 fs-4 fw-bold">{detour.name}</div>
```

Comment:
`fw-bold` is a touch too bold. Design: font-weight 600, while fw-bold is 700

Reply:
[Oops](https://github.com/mbta/skate/pull/2770/commits/b86438962d9fcc4a8d6f0f757148b916431c7357)